Assets/Scripts/UserInterface/MainMenu.cs

```diff
@@ -25,11 +25,11 @@ public void LoadCredits()
 
     public void QuitGame()
     {
-#if UNITY_EDITOR
-        UnityEditor.EditorApplication.isPlaying = false;
-#else
-        Application.Quit();
-#endif
+        #if UNITY_EDITOR
+            UnityEditor.EditorApplication.isPlaying = false;
+        #else
+            Application.Quit();
+        #endif
```

Comment:
iirc, preprocessor directives should be to the most left column.

Reply:
Fixed